scripts/pre_commit_linter.py

```diff
@@ -535,6 +535,9 @@
     '/scripts/install_third_party_libs.py',
     '/.github/']
 
+# Check if project is running in CircleCI.
+IS_CI = os.getenv('CIRCLECI')
```

Comment:
Do you need to wrap `bool()` around this?

Reply:
I will wrap bool() around this in the next commit.